login-workflow/example/src/app/app.routing.ts

```diff
@@ -1,13 +1,18 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-import { authSubRoutes, PxbAuthGuard, AUTH_ROUTE } from '@pxblue/angular-auth-workflow';
+import { getAuthSubRoutes, PxbAuthGuard, AUTH_ROUTES } from '@pxblue/angular-auth-workflow';
 import { HomeComponent } from './pages/home/home.component';
 import { AuthComponent } from './pages/auth/auth.component';
 import { DashboardComponent } from './pages/dashboard/dashboard.component';
 
+// The default workflow routes can be overwritten if needed.
+AUTH_ROUTES.AUTH_WORKFLOW = 'auth';
+AUTH_ROUTES.CONTACT_SUPPORT = 'assistance';
+
+const authWorkflowRoutes = getAuthSubRoutes();
 const routes: Routes = [
-    { path: '', redirectTo: AUTH_ROUTE, pathMatch: 'full' },
-    { path: AUTH_ROUTE, component: AuthComponent, children: authSubRoutes },
+    { path: '', redirectTo: AUTH_ROUTES.AUTH_WORKFLOW, pathMatch: 'full' },
+    { path: AUTH_ROUTES.AUTH_WORKFLOW, component: AuthComponent, children: authWorkflowRoutes },
```

Comment:
Yep the auth guard will still work because all it does is use the `SecurityContext` object to redirect to the "/" path if a user isn't logged in.

I just noticed an issue in our default `AUTH_ROUTES.ON_AUTHENTICATED` value though.  If a user tries to directly goes to `/dashboard` they are redirected after logging in to the `/home` page instead.  
This only happens with the empty `AUTH_WORKFLOW` route, so I'll fix this now.

Reply:
I've refactored the route-matching logic used inside the ng workflow.  

The ng auth workflow's default `ON_AUTHENTICATED` value will be any non auth-related route.